src/components/global/footer.tsx

```diff
@@ -251,8 +271,33 @@ const StyledSlideText = styled.span`
   }
 `;
 
+const StyledPillIconButton = styled(PillIconButton)`
```

Comment:
After adding styles from above into this, change it to be the following

```suggestion
const StyledPillIconButton = styled(OriginalPillIconButton)`
```